packages/core/src/controllers/verify.ts

```diff
@@ -47,21 +61,35 @@ export class Verify extends IVerify {
 
   public resolve: IVerify["resolve"] = async (params) => {
     if (this.isDevEnv) return "";
+    const mainUrl = params?.verifyUrl || VERIFY_SERVER;
```

Comment:
if the user defines a `verifyUrl` by param, should we use the fallback in case of failure too? Or in that case just fail without a fallback

Reply:
atm is an immediate fallback to `.org` consistently to `relayUrl` fallback